src/kbmod_wf/task_impls/kbmod_search.py

```diff
@@ -14,3 +19,70 @@ def kbmod_search(input_wu=None, result_file=None, logger=None):
         f.write(f"Logged: {value} - {time.time()}\n")
 
     return result_file
+
+
+def kbmod_search(
+    wu_filepath: str = None,
+    search_config_filepath: str = None,  # ! Seems like this should be provided in the runtime_config dictionary ???
+    result_filepath: str = None,
+    runtime_config: dict = {},
+    logger: Logger = None,
+):
```

Comment:
A short docstring would also be nice for this phase

Reply:
Done